pkg/api/app_blocks.go

```diff
@@ -9,12 +10,12 @@ type Score struct {
 	Score string `json:"score"`
 }
 
-func (a *App) BlocksScoreAt(at proto.Height) (*Score, error) {
+func (a *App) BlocksScoreAt(at proto.Height) (Score, error) {
```

Comment:
I don't think here should be a function for just `score, err := a.state.ScoreAtHeight(at)`

Reply:
I don't think so because this layer hides node and state internals.